assembl/static2/js/app/pages/idea.jsx

```diff
@@ -54,8 +54,35 @@ const noRowsRenderer = () => {
 };
 
 class Idea extends React.Component {
+  topPostsToLoad = (topPosts, edges) => {
+    const { hash } = window.location;
+    if (hash !== '') {
+      const id = hash.replace('#', '');
+      const post = edges.filter((edge) => {
+        return edge.node.id === id;
+      });
+      if (post.length === 0) {
+        return null;
+      }
+      let topPostId = post[0].node.parentId;
+      if (!topPostId) {
+        topPostId = post[0].node.id;
+      }
+      const index = topPosts
+        .map((value) => {
+          return value.id;
+        })
+        .indexOf(topPostId);
```

Comment:
`topPosts.findIndex(value => value.id == topPostId)` does the job here, no ? (I haven't tested it)

Reply:
yeah, it should do it